hcl2template/testdata/complete/build.pkr.hcl

```diff
@@ -1,4 +1,33 @@
 
+packer {
```

Comment:
nice. Can we add an invalid file that tests to make sure if there are duplicates we error?

Reply:
Done !